docs/standard/garbage-collection/unmanaged.md

```diff
@@ -31,15 +31,15 @@ If your types use unmanaged resources, you should do the following:
   - Define a [finalizer](../../csharp/programming-guide/classes-and-structs/finalizers.md). Finalization enables the non-deterministic release of unmanaged resources when the consumer of a type fails to call <xref:System.IDisposable.Dispose%2A?displayProperty=nameWithType> to dispose of them deterministically.
 
     > [!WARNING]
-    > However, because object finalization can be a complex and an error-prone operation, we recommend that you use a safe handle instead of providing your own finalizer.
+    > Object finalization can be a complex and error-prone operation, we recommend that you use a safe handle instead of providing your own finalizer.
 
 Consumers of your type can then call your <xref:System.IDisposable.Dispose%2A?displayProperty=nameWithType> implementation directly to free memory used by unmanaged resources. When you properly implement a <xref:System.IDisposable.Dispose%2A> method, either your safe handle's <xref:System.Object.Finalize%2A> method or your own override of the <xref:System.Object.Finalize%2A?displayProperty=nameWithType> method becomes a safeguard to clean up resources in the event that the <xref:System.IDisposable.Dispose%2A> method is not called.
 
 ## In this section
 
 [Implementing a Dispose method](implementing-dispose.md) describes how to implement the dispose pattern for releasing unmanaged resources.
 
-[Using objects that implement `IDisposable`](using-objects.md) describes how consumers of a type ensure that its <xref:System.IDisposable.Dispose%2A> implementation is called. We recommend using the C# `using` (or the Visual Basic `Using`) statement to do this.
+[Using objects that implement `IDisposable`](using-objects.md) describes how consumers of a type ensure that its <xref:System.IDisposable.Dispose%2A> implementation is called. We strongly recommend using the C# `using` (or the Visual Basic `Using`) statement to do this.
```

Comment:
@BillWagner Do you think this new adverb is justified?

Reply:
I'd probably leave it off. The general convention is to prefer `using` when you wouldn't have a `try` - `catch` for cases where recovery is possible.  If the code would include `try` - `catch` - `finally`, writing the `finally` clause is likely cleaner.

Reply:
I agree with Bill, there are times when we recommend not having implementations in a `using` too.